server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -300,23 +300,35 @@ public void run() {
           }
         }), 5000, 5000, TimeUnit.MILLISECONDS);
 
-    final long walogMaxSize = aconf.getAsBytes(Property.TSERV_WALOG_MAX_SIZE);
-    final long walogMaxAge = aconf.getTimeInMillis(Property.TSERV_WALOG_MAX_AGE);
+    @SuppressWarnings("deprecation")
+    final long walMaxSize =
+        aconf.getAsBytes(aconf.resolve(Property.TSERV_WAL_MAX_SIZE, Property.TSERV_WALOG_MAX_SIZE));
+    @SuppressWarnings("deprecation")
+    final long walMaxAge = aconf
+        .getTimeInMillis(aconf.resolve(Property.TSERV_WAL_MAX_AGE, Property.TSERV_WALOG_MAX_AGE));
     final long minBlockSize =
         context.getHadoopConf().getLong("dfs.namenode.fs-limits.min-block-size", 0);
-    if (minBlockSize != 0 && minBlockSize > walogMaxSize) {
+    if (minBlockSize != 0 && minBlockSize > walMaxSize) {
+      @SuppressWarnings("deprecation")
+      Property deprecatedProp = Property.TSERV_WALOG_MAX_SIZE;
       throw new RuntimeException("Unable to start TabletServer. Logger is set to use blocksize "
-          + walogMaxSize + " but hdfs minimum block size is " + minBlockSize
-          + ". Either increase the " + Property.TSERV_WALOG_MAX_SIZE
+          + walMaxSize + " but hdfs minimum block size is " + minBlockSize
+          + ". Either increase the " + aconf.resolve(Property.TSERV_WAL_MAX_SIZE, deprecatedProp)
```

Comment:
This message can be changed to only refer to the new property. We don't need to recommend the old one just because they were previously using it.